.github/workflows/reposchutz.yml

```diff
@@ -0,0 +1,40 @@
+name: repository
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize, labeled, unlabeled]
+
+jobs:
+  check:
+    name: Protection checks
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    env:
+      GIT_DIR: git-dir.git
+    steps:
+      - name: Clear .github-changes label
+        if: ${{ !endsWith(github.event.action, 'labeled') }}
+        uses: actions/github-script@v3
+        with:
+          script: |
+            try {
+              const { repo: { owner, repo }, number: issue_number } = context;
+              await github.issues.removeLabel({ owner, repo, issue_number, name: '.github-changes' });
+            } catch (e) {
+              if (e.name == 'HttpError' && e.status == 404) {
+                /* expected: 404 if label isn't set */
+              } else {
+                throw e;
+              }
+            }
+
+      - name: Check for .github changes
+        # We want to run this check any time the .github-changes label is not
+        # set, which needs to include the case where we just unset it above.
+        if: ${{ !endsWith(github.event.action, 'labeled') ||
+                !contains(github.event.pull_request.labels.*.name, '.github-changes') }}
```

Comment:
I find a label with a dot a bit weird -- can we drop that?

Reply:
Why?  The github docs even mention that label names could happily contain emoji.  Do you have a suggestion for an alternative?

Reply:
It's not related to what the API allows, this was a purely aesthetical remark. Just "github-changes"? But this is of course not that important, not a strong opinion.

Reply:
I'd prefer to leave it as is, in that case.  ".github" is the thing that's changing, so it makes sense to me to name it as such :)